.npmrc

```diff
@@ -1,2 +1,4 @@
 save-exact = true
 tag-version-prefix=""
+auto-install-peers = true
```

Comment:
Was the previous yarn behavior installing peers? I thought it didn't?

Reply:
Ah yeah this is left over from debugging good catch! Removed in https://github.com/vercel/next.js/pull/37259/commits/11cd69f0a3913cbe3862b68def71fd7c76cb30ff

Reply:
Hmm it seems yarn was auto installing peer deps before as updating a package without this does break the dependency tree